packages/libs/react-ui/src/components/Tag/Tag.stories.tsx

```diff
@@ -1,62 +1,129 @@
-import { Button } from '@components/Button';
-import type { ITagProps } from '@components/Tag';
-import { Tag } from '@components/Tag';
```

Comment:
Don't we still want to add this component as well to our Storybook?

Reply:
This isn't a common use case, so I didn't, but I can add it if you think it's beneficial?

Reply:
I wanted to see how this component looks since it is different from the ones being added to the storybook right now. It might be beneficial, but I'll let you decide 😁

Reply:
I'll add it 👍